1-js/11-async/01-callbacks/article.md

```diff
@@ -214,14 +214,14 @@ loadScript('1.js', function(error, script) {
 });
 ```
 
-In the code above:
-1. We load `1.js`, then if there's no error.
-2. We load `2.js`, then if there's no error.
-3. We load `3.js`, then if there's no error -- do something else `(*)`.
+Trong đoạn mã trên:
+1. Ta nạp tệp `1.js`, sau đó không có lỗi trả về.
+2. Ta nạp tệp `2.js`, sau đó không có lỗi trả về.
+3. Ta nạp tệp `3.js`, sau đó không có lỗi trả về -- ta làm hành động gì đó trong else `(*)`.
 
-As calls become more nested, the code becomes deeper and increasingly more difficult to manage, especially if we have a real code instead of `...`, that may include more loops, conditional statements and so on.
+Như cách gọi gọi lồng nhiều cấp trên, đoạn code sẽ trở nên phức tạp hơn để quản lý chúng, đặc biệt nếu ta thực tại ta cần xử lý nhiều hơn ở các chỗ `...`, các chỗ đó có thể nhiều vòng lặp hoặc các điều kiện và hơn thế.
```

Comment:
especially if we have a real code instead of `...`, that may include more loops, conditional statements and so on.

should be translated 

đặc biệt nếu ta có nhiều code hơn thay vì `...`, nó có thể chứa nhiều vòng lặp hơn, nhiều điều kiện hơn và hơn thế nữa.